login.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+
+<link rel="stylesheet" type="text/css" href="css/login.css">
+
+</head>
+<body>
+
+<div id="external">
+<div id="login">
+	<h4>Interview Feedback</h4>
+	<form method="POST">
```

Comment:
Hmmm. Nice! But what does this `method="POST"` actually do? And where does this `<form>` actually lead? What API endpoint does it communicate with?